cmd/brew-services.rb

```diff
@@ -236,6 +237,33 @@ def restart(target)
       end
     end
 
+    # "load" a plist
+    def launchctl_load(plist, _function, service)
+      if root?
+        domain_target = "system"
+      else
+        domain_target = "gui/#{Process.uid}"
+      end
+
+      if MacOS.version < :yosemite
+        # This syntax was deprecated in Yosemite
+        safe_system launchctl, "load", "-w", plist
+      else
+        # New syntax has improved errr-handling.
+        safe_system launchctl, "enable", "#{domain_target}/#{service.label}"
+        if $?.to_i.nonzero?
```

Comment:
`safe_system` will fail with an error so this failure check will never be hit (I realise the previous code also made this mistake).